src/SFA.DAS.EmployerIncentives.Database/Scripts/Update.ApprenticeshipIncentives.sql

```diff
@@ -3,3 +3,16 @@
 FROM incentives.ApprenticeshipIncentive AI
 INNER JOIN IncentiveApplicationApprenticeship IAA ON IAA.Id = AI.IncentiveApplicationApprenticeshipId
 WHERE AI.UKPRN IS NULL
+GO
+UPDATE i SET 
```

Comment:
Would the DACPAC deploy support converting PausePayments into a computed column, to save having to populate the same data in two fields?